articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,384 @@
+    MSSQLSERVER_HOST=localhost \
+    MSSQLSERVER_PORT=1433 \
+    java -jar ./target/todo-list-bootable.jar -Djboss.node.name=node2 -Djboss.socket.binding.port-offset=10
+    ```
+
+    > [!NOTE]
+    > By default the Bootable JAR configures the JGroups subsystem to use the UDP protocol and sends messages to discover other cluster members to the 230.0.0.4 multicast address. To properly verify the clustering capabilities on your local machine, your Operating System should be capable of sending and receiving multicast datagrams and route them to the 230.0.0.4 IP through your ethernet interface. Check your network configuration and verify this is working if you see warnings related to the cluster on the server logs.
+
+1. Open `http://localhost:8080/` in your browser to visit the application home page. If you have created more instances, you can access them by shifting the port number, for example `http://localhost:9080/`. The application will look similar to the following image:
+
+    ![ToDo EAP demo Application](./media/howto-deploy-java-eap-app/todo-demo-application.png)
+
+1. Check the application health endpoints. These endpoints will be used by OpenShift to verify when your pod is live and ready to receive user requests:
+
+   ```bash  
+    todo-list (bootable-jar) $ curl http://localhost:9990/health/live
+    {"status":"UP","checks":[{"name":"SuccessfulCheck","status":"UP"}]}
+    todo-list (bootable-jar) $ curl http://localhost:9990/health/ready
+    {"status":"UP","checks":[{"name":"deployments-status","status":"UP","data":{"todo-list.war":"OK"}},{"name":"server-state","status":"UP","data":{"value":"running"}},{"name":"boot-errors","status":"UP"},{"name":"DBConnectionHealthCheck","status":"UP"}]}
+    ```
+
+1. Press **Control-C** to stop the application.
+1. Execute the following command to stop the database server:
+
+    ```bash
+    docker stop mssqlserver
+    ```
+
+1. If you don't plan to use the docker database again, execute the following command to remove the database server from your docker registry:
+
+    ```bash
+    docker rm mssqlserver
+    ```
+
+## Deploy to OpenShift
+
+Before deploying the demo application on OpenShift we will deploy the database server. The database server will be deployed by using a [DeploymentConfig OpenShift API resource](https://docs.openshift.com/container-platform/4.8/applications/deployments/what-deployments-are.html#deployments-and-deploymentconfigs_what-deployments-are). The database server deployment configuration is available as a YALM file on the application source code. To deploy the application, we are going to use the JBoss EAP Helm Charts that are already available in ARO. We also need to supply the desired configuration, for example, the database user, the database password, the driver version we want to use, and the connection information used by the data source. Since this information contains sensible information, we will use OpenShift secrets objects to store it.
```

Comment:
I think the part beginning with "To deploy the application" should be a new paragraph.